### ISSUE TITLE
sweden banned config doesnt show it as banned

### ISSUE THREAD

**Issue by eero619** [Xephi/AuthMeReloaded#900]
http://hastebin.com/nokuwiporu.vhdl

there is the config and it does not say any countries banned but sweden is banned.

**Comment by ljacqu** [comment]
Let's make sure AuthMe is causing this first. 
- What's the message a user gets?
- Does this problem go away when AuthMe is disabled?

**Comment by Twonox** [comment]
I have tested and I have the same problem (but I don't know if sweden is banned or no).
We must put anything in countries list to it works:
```
    countries:
    - 'XYZ'
```

If we put `[]` it doesn't work.

**Comment by sgdc3** [comment]
@ljacqu very strange O_o

**Comment by ljacqu** [comment]
Weird, just tried with config posted by OP and can't reproduce. @eero619 What do you get when you do `/authme version`?
@Twonox are you sure you didn't mean `countriesBlacklist`? If `countries` has some "impossible" value I'd expect no one be able to join anymore (confirmed behavior locally).

**Comment by Twonox** [comment]
I mean `countries`, I'm sure.
If `enableProtection: true` with `countries: []`, no banned countries.

**Comment by ljacqu** [comment]
Oh, I understood you the wrong way around. It makes sense like this.

**Comment by Twonox** [comment]
But I don't know why Sweden is banned.

> there is the config and it does not say any countries banned but sweden is banned.

**Comment by eero619** [comment]
authme version: authmereloaded v5.2-BETA3 build 1134
Friend sent me a picture in the server list it says your country is banned and kicks with same message.
now it cant be the antiproxyplugin bc when i try with proxy it says restart your router with kick.
plugins:
http://prntscr.com/c5a0zk
and it can only be caused by authme bc otherwise why would something look like this:
http://prntscr.com/c5a279

im going to see what it does when i write 
 countries:
 - 'XYZ'
anyways if you have any ideas contact me

**Comment by eero619** [comment]
just making sure evrybody understands i want antibot system to work and i want 0 banned countries i also went into the java file and edited the config to be exactly the same as it is here what i posted so please help me, both configs inside the plugin and also in the folder of the plugin are the same no worries.

**Comment by eero619** [comment]
also i did ask my hoster and they said we have 0 banned countries it has to be something with your plugins.

**Comment by ljacqu** [comment]
Could you update to a newer build? That would be fantastic. We had some issues with countries a while ago, and b1134 is quite a ways back :) 
http://ci.xephi.fr/job/AuthMeReloaded/

**Comment by eero619** [comment]
ok i will i update when i can

**Comment by eero619** [comment]
wich one do i take spigot if i have spigot or what

**Comment by ljacqu** [comment]
legacy for Minecraft 1.7.10 and Cauldron, otherwise spigot

**Comment by eero619** [comment]
il let you ppl know if the build fixed as soon as somebody can tell me that sweden i banned or fine

**Comment by ljacqu** [comment]
Looking forward to hearing back from you, @eero619

**Comment by Twonox** [comment]
@eero619  If you want only Swedish in your server you must put SE.
```
countries:
- 'SE'
```

**Comment by eero619** [comment]
sorry new issue not sure yet about sweden tough people after logging in sometimes not always cant move at all even deisntalling authme after this occurs doesnt remove the effect pls help !!!!

**Comment by ljacqu** [comment]
Please use a site like http://hastebin.com to paste configs, the scrolling has become unreal in this thread :P The speed thing is being discussed in #890, see also the comment from rErEaT there to restore speed for now (we're working on a permanent solution).

**Comment by eero619** [comment]
ok next time i will, just i fixed it for temporary but i have no clue how to unfreeze people pls tell me that bc they can only move while jumping. pls tell me. and i do have a new config :D here it is:
http://hastebin.com/mexoxajoto.vhdl
i changed alot of things but still cant get ppl to move when they have the freeze or whatever.

**Comment by ljacqu** [comment]
https://github.com/Xephi/AuthMeReloaded/issues/890#issuecomment-239596517

Any news about Sweden?

**Comment by eero619** [comment]
sorry he hasnt been on we will get news soon someday idk i update when its fixed with current config then or otherwise not fixed and config :+1:

**Comment by ljacqu** [comment]
Thanks; if the issue is still there please reopen this issue, or drop a comment (not sure when Github allows someone to reopen an issue).

**Comment by eero619** [comment]
Still sweden is banned config
hastebin link:
http://hastebin.com/qekucavoni.vhdl

**Comment by Twonox** [comment]
Replace `contries: []` by
    countries:
    - 'SE'

**Comment by ljacqu** [comment]
@eero619 Please download the JAR (legacy or spigot) from https://github.com/AuthMe/AuthMeReloaded/tree/900-country-protection-debug/samples, and in your config:
```yml
logLevel: 'DEBUG'
```
and keep `logConsole: true` as you have it now.

Please don't do what @Twonox suggests—that would _only_ allow Sweden from joining.

**Comment by Twonox** [comment]
@eero619 You have `enableProtection: true` If you won't banned contries, just set it to `false`. If you would antibot, you can use authme's AntiBot or keep `enableProtection: true` but with a whitelist in `countries: []`.

**Comment by Twonox** [comment]
@ljacqu eero619 want AntiBot but he don't know how `enableProtection` works.

**Comment by eero619** [comment]
hastebin link:
http://hastebin.com/foxaboxoqi.coffee
so umm idk what this is about but i downloaded the spigot one @ljacqu and idk if it checked anything.
Can it be this plugin what causes the banned country: AntiJoinBot - https://dev.bukkit.org/bukkit-plugins/easy-anti-join-bot-proxie/ mabe but im not sure.

and that got me thinking if i disableprotection does antibot authmes one still work ? and if it does cant i just stay on my version of the authmes plugin, disable protection - enableprotection: false and still stay protected from bots with authme antibot and antijoinbot.
Not sure if you can help me with AntiJoinBot.

So in simple terms if i disableprotection does antibot authmes one disable itself ?
and if it does can i make them seperate so i have no banned countires with disabledprotection and have antibot authmes one still working ? pls tell me.

**Comment by Twonox** [comment]
The cause of your problem is probably Authme, not AntiJoinBot (but, as I remember, anitjoinbut isn't a good protection).
Authme's AntiBot and Authme's ContriesProtection are two different and separate protection against bots.
`enableAntiBot: true` blocks new players to join if there is a lot of new players in 5sec.
`enableProtection: true` ban contries, but you can add whitelisted countries code like SE to not ban Sweden (in `contries: []`)
In your screen (http://prnt.sc/c5a279) we can see Authme's message for banned counties. So, if you set `enableProtection: false` normally there is no banned contries. Please set it to `false` and tell we if Sweden is still banned.

**Comment by eero619** [comment]
sweden is still banned new config:
http://hastebin.com/guxahetudi.vhdl

**Comment by ljacqu** [comment]
did you really restart with the new config? it's kind of weird that you wouldn't be able to at least disable the country protection... You're using an old build so maybe the best course of action at this point is for us to fix the issue linked above so you can upgrade to a newer build.

**Comment by eero619** [comment]
Give me the newest build for spigot 1.10
current authme version:
authmereloaded v5.2 snapshot build 1287
help me :D

**Comment by Twonox** [comment]
AuthMe-5.2-SNAPSHOT-spigot.jar

**Comment by eero619** [comment]
legacy or this https://dev.bukkit.org/bukkit-plugins/authme-reloaded/files/79-auth-me-5-2-beta-3/

**Comment by ljacqu** [comment]
> legacy for Minecraft 1.7.10 and Cauldron, otherwise spigot [[x]](https://github.com/Xephi/AuthMeReloaded/issues/900#issuecomment-239631626)

Please use latest build from http://ci.xephi.fr/job/AuthMeReloaded/, as Twonox posted

**Comment by ljacqu** [comment]
@eero619 , the issue you experienced with the new version (#907) has been fixed by @DNx5,
if I could ask you again to download the appropriate JAR from https://github.com/AuthMe/AuthMeReloaded/tree/900-country-protection-debug/samples we will see debug output relating to country protection, which would be helpful. Thanks

**Comment by eero619** [comment]
well this sucks the guy blocked me who i was contacted with from sweden anyways i installed the last plugin what you guys told me so i hope its fixed but we will not have a clue until somebody from sweden contacts me. anyways you can close this thread i hope its fixed.

**Comment by sgdc3** [comment]
Thank you, if it happens again just poke us ;)

**Comment by eero619** [comment]
i think its fixed. 
https://github.com/AuthMe/AuthMeReloaded/tree/900-country-protection-debug/samples
fixed it for me :+1: 

proof:
http://prntscr.com/c9t7jv

**Comment by sgdc3** [comment]
Ok, now you can use latest build again to remove the debug messages ;)
